tests/sagemaker-tests/test_sagemaker_utils.py

```diff
@@ -0,0 +1,193 @@
+"""
+    Copyright 2025 Contributors
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+       http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+"""
+import os
+import tempfile
+import tarfile
+import pytest
+from launch_utils import wrap_model_artifacts
+
+
+def create_dummy_file(file_path):
+    """ Create an empty dummy file for testing
+    """
+    if os.path.exists(file_path):
+        return
+
+    with open(file_path, 'w') as f:
+        f.close()
+
+def test_wrap_model_artifacts():
+    """
+    """
+    # test case 1: normal case, everything is given and the tar file created.
+    with tempfile.TemporaryDirectory() as tmpdirname:
+        entry_path = os.path.join(tmpdirname, 'nc_infer_entry.py')
+        model_path = os.path.join(tmpdirname, 'model.bin')
+        yaml_path = os.path.join(tmpdirname, 'test.yaml')
+        json_path = os.path.join(tmpdirname, 'test.json')
+
+        create_dummy_file(entry_path)
+        create_dummy_file(model_path)
+        create_dummy_file(yaml_path)
+        create_dummy_file(json_path)
+
+        output_path = os.path.join(tmpdirname, 'output_folder')
+        os.makedirs(output_path, exist_ok=True)
```

Comment:
You also need to test the case when output_path does not exist.

Reply:
add a new test case.